src/main/kotlin/no/nav/syfo/behandler/kafka/dialogmeldingfrombehandler/KafkaDialogmeldingFromBehandler.kt

```diff
@@ -84,3 +103,52 @@ private fun getPartnerId(fellesformatXML: String, navLogid: String, msgId: Strin
         null
     }
 }
+
+private fun getIdenterForBehandler(
+    fellesformatXML: String,
+    navLogid: String,
+    msgId: String,
+): Map<BehandleridentType, String> {
+    return try {
+        val xmlBehandler = getObjectFromXmlString<XMLHealthcareProfessional>(
+            fellesformatXML,
+            "HealthcareProfessional"
+        )
+        val behandlerIdenter = xmlBehandler.ident
+
+        behandlerIdenter.associate {
+            BehandleridentType.valueOf(it.typeId.v) to it.id
+        }
+    } catch (e: Exception) {
+        log.warn("Can't find behandleridenter in dialogmelding from behandler: navLogId: $navLogid, msgId: $msgId", e)
+        emptyMap()
+    }
+}
+
+private fun updateDialogmeldingEnabledForKontor(
+    partnerId: PartnerId,
+    database: DatabaseInterface,
+) {
+    val dialogmeldingEnabled = database.updateBehandlerKontorDialogmeldingEnabled(partnerId)
```

Comment:
Denne oppdaterer hvert gang selv om tidsstempelet/flagget allerede er satt, ikke sant? Hadde det vært en ide å bare sette det der det mangler? (da ville vi se i databasen når et kontor ble enabled)

Reply:
Jeg kan evt. ta det i en egen oppgave, siden dette egentlig ikke er nytt, bare flyttet litt på. Men ønsker vi ikke å sette det hver gang, sånn at vi har muligheten til å hente kontor som er bekreftet f.eks. siste x måneder?

Reply:
Jo, man kan argumentere for det også, så vi kan tenke litt mer på det.

Reply:
Hva med et eget last_used felt elns?